target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/vr9_avm_fritz7430.dts

```diff
@@ -0,0 +1,275 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7430", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7430";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_info_red;
+		led-running = &led_power;
+		led-upgrade = &led_info_green;
+
+		led-wifi = &led_wifi;
```

Comment:
I've seen the discussion about the Info LED above. Using the INFO LED for DSL connection state is valid for this series, as outlined in https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=3032bf7f89e8ad5b69ab2f031287f475e5d2b829 . Apart from that, LGTM.

Reply:
Okay, that way it's at least consistent, so I'm fine with it.

Reply:
Added the DSL LED again.